test/conftest.py

```diff
@@ -40,7 +40,7 @@ def make_container(service_cls, config):
     for c in all_containers:
         try:
             c.stop()
-        except:
+        except:  # noqa: E722
```

Comment:
I think you want `except: Exception` here. Can't see a reason why we should be catching anything else.

Also, I wonder why we are reimplementing the `container_factory` here. Does `nameko-sqlalchemy` require `stop` instead of `kill` for teardown?

Reply:
@mattbennett You're right about the exception and we do not seem to require `stop` instead of `kill` either. I have removed the fixture and all tests are passing without reimplementing it 👍

Reply:
Nice! I poked around in the history and it turns out that this was ported from Nameko core before Nameko had its own pytest plugin, so that explains the need for reimplementation.

Reply:
That makes sense. Good to remove old code :)